website/docs/concepts/filter_service/README.md

```diff
@@ -0,0 +1,93 @@
+---
+id: filter-overview
+title: Filter
+---
+
```

Comment:
Great overview -- I only think it is only missing information about requiring a connection to an RTK base station

Reply:
Addressed in commit da3ef391